cli/util/uploader.go

```diff
@@ -425,14 +425,15 @@ func uploadIndexFiles(ctx *cli.Context, p *pool.Pool, containerID cid.ID, accoun
 			for idx := range indexFileSize {
 				if _, ok := processedIndices.Load(idx); !ok {
 					count++
+					fmt.Fprintf(ctx.App.Writer, "Index file %d: fetching missing block %d\n", i, i*indexFileSize+idx)
```

Comment:
BTW, won't it be too chatty?

Reply:
It will, but currently there's completely no way to track the progress of "gaps" completion. Everything what I have is just one line:
```
2024-11-05 13:39:11.228	Index file X generated, checking for the missing blocks...
``` 
which is hanging in the air for several hours. This log allows us to track the progress and (in bad cases) to learn which objects are missing from the initial SEARCH request, it may help during NeoFS-related issues debugging.

Reply:
Hence I suggest to keep these logs for now.